projects/pgai/pgai/_install/install.py

```diff
@@ -11,6 +12,10 @@
 
 log = structlog.get_logger()
 
+OLD_AI_EXTENSION_EXCEPTION = Exception(
+    "You have an old version of the ai extension installed. You must upgrade the ai extension to version 0.10.0 or greater before installing pgai"  # noqa: E501
```

Comment:
I would suggest that we include the old version in this error message, something like:

```suggestion
    "The ai extension is outdated and must be upgraded. Installed version: {version}, required version: {required_version}" # noqa: E501
```

This exception has one usage site, so I would suggest inlining it there.